### PR TITLE
fix: set fixed versions requirements except for jina encoders part 2

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: encoders/nlp/LaserEncoder/aux.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import pkg_resources
+
+def test(requirements_path):
```
Comment (cristianmtr) [inline encoders/nlp/LaserEncoder/aux.py:3]: I'm not sure I understand how this would be used. Is it part of CI, or is it up to the dev to run it?

Comment (cristianmtr) [reply encoders/nlp/LaserEncoder/aux.py:3]: Why are we setting that version? Can't we just do it in the requirements?

Comment (JoanFM) [reply encoders/nlp/LaserEncoder/aux.py:3]: oh this is a file that should not have been uploaded will remove

File: encoders/nlp/FarmTextEncoder/requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
-farm
+farm==0.5.0
```
Comment (florian-hoenicke) [inline encoders/nlp/FarmTextEncoder/requirements.txt:1]: can optionally be set to `0.6.0`. The tests pass.

**Review by florian-hoenicke — DISMISSED** [review]

File: encoders/nlp/UniversalSentenceEncoder/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-protobuf>=3.9.2
-grpcio>=1.24.3
-tensorflow
-tensorflow_hub
\ No newline at end of file
+protobuf==3.14.0
+grpcio==1.34.0
```
Comment (maximilianwerk) [inline encoders/nlp/UniversalSentenceEncoder/requirements.txt:2]: Why are they sometimes removed and sometimes bumped? Shouldn't they come from core directly?

Comment (JoanFM) [reply encoders/nlp/UniversalSentenceEncoder/requirements.txt:2]: when they are removed is because some tests were failing and dig deeper to fix it.

Comment (JoanFM) [reply encoders/nlp/UniversalSentenceEncoder/requirements.txt]: trying to remove noew

**Pull request closed without merging**